### PR TITLE
EWPP-4496: Add Introduction field to various patterns.

### PR DESCRIPTION
**Pull request by intelektron**
### Description

Add Introduction field to link lists.

### CODE REVIEW

File: templates/compositions/ec-component-highlighted-news-block/highlighted-news-block.html.twig [hunk 1]
```diff
@@ -19,6 +20,14 @@
         </div>
       </div>
     {% endif %}
+    {% set _introduction = introduction|default('') %}
```
Comment (22Alexandra) [inline templates/compositions/ec-component-highlighted-news-block/highlighted-news-block.html.twig]: Why is this needed? Instead of moving the value from a variable to another, we can just check if `introduction` is not empty.

**Review by 22Alexandra — CHANGES_REQUESTED** [review]
We need test coverage for all the patterns that have been changed.
Please check `tests/src/Kernel/fixtures/rendering.yml`

File: templates/compositions/ec-component-highlighted-news-block/highlighted-news-block.html.twig (hunk 1 above)
Comment (intelektron) [reply templates/compositions/ec-component-highlighted-news-block/highlighted-news-block.html.twig]: I just followed the way from other components, I can change it.

Comment (intelektron) [reply templates/compositions/ec-component-highlighted-news-block/highlighted-news-block.html.twig]: Fixed

**Comment by intelektron** [comment]
> We need test coverage for all the patterns that have been changed. Please check `tests/src/Kernel/fixtures/rendering.yml`

I've added the introduction field to pattern fixtures

**Review by 22Alexandra — DISMISSED** [review]
